src/main/java/seedu/address/model/participant/BirthDate.java

```diff
@@ -15,7 +15,7 @@
  */
 public class BirthDate {
 
-    public static final String MESSAGE_DATE_CONSTRAINTS = "Date of birth cannot be in the future or invalid";
+    public static final String MESSAGE_DATE_CONSTRAINTS = "Date of birth cannot be in the future or invalid date given";
```

Comment:
For this, maybe can revert the changes here, I think the checks I do and additional messages should do the trick